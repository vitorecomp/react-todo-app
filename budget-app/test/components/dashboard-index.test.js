import React from 'react'
import { shallow } from 'enzyme'

// import DashboardIndex from '../../src/components/dashboard-index'



test('should render dashboard conrrectly', () => {
	// const wrapper = shallow(<DashboardIndex />) 
	// expect(wrapper).toMatchSnapshot();
})
