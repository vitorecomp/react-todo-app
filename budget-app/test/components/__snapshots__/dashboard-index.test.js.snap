// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not-found conrrectly 1`] = `
<div>
  <h1>
    404!
  </h1>
  <Link
    to="/"
  >
     Go Home
  </Link>
</div>
`;
